### ISSUE TITLE
Call to undefined method `ParserOutput::addModuleScripts()`

### ISSUE THREAD

**Issue by kghbln** [oetterer/BootstrapComponents#15]
### Setup
- MediaWiki | 1.34.0-rc.1 (ccb2bb2)19:12, 18 November 2019
- PHP | 7.2.24-0ubuntu0.18.04.1 (apache2handler)
- MariaDB | 10.1.41-MariaDB-0ubuntu0.18.04.1
- Semantic MediaWiki | 3.2.0-alpha (aef1b47) 18:20, 18 November 2019
- Bootstrap | 1.3.0 (ea33f24) 09:57, 15 January 2019
- BootstrapComponents | 1.2.3 (ee28412) 18:23, 9 September 2019

### Issue
When rebuilding the wiki with "rebuildData.php" I get the following errot:
```
Error from line 208 of /../w/extensions/BootstrapComponents/src/ParserOutputHelper.php: Call to undefined method ParserOutput::addModuleScripts()
```
### Backtrace
```
#0 /../w/extensions/BootstrapComponents/src/AbstractComponent.php(310): BootstrapComponents\ParserOutputHelper->loadBootstrapModules()
#1 /../w/extensions/BootstrapComponents/src/AbstractComponent.php(332): BootstrapComponents\AbstractComponent->augmentParserOutput()
#2 /../w/extensions/BootstrapComponents/src/AbstractComponent.php(158): BootstrapComponents\AbstractComponent->initComponentData(BootstrapComponents\ParserRequest)
#3 /../w/extensions/BootstrapComponents/src/Hooks/ParserFirstCallInit.php(159): BootstrapComponents\AbstractComponent->parseComponent(BootstrapComponents\ParserRequest)
#4 /../w/includes/parser/Parser.php(4293): BootstrapComponents\Hooks\ParserFirstCallInit->BootstrapComponents\Hooks\{closure}(string, array, Parser, PPFrame_Hash)
#5 /../w/includes/parser/PPFrame_Hash.php(328): Parser->extensionSubstitution(array, PPFrame_Hash)
#6 /../w/includes/parser/Parser.php(3330): PPFrame_Hash->expand(PPNode_Hash_Tree, integer)
#7 /../w/includes/parser/Parser.php(1489): Parser->replaceVariables(string)
#8 /../w/includes/parser/Parser.php(593): Parser->internalParse(string)
#9 /../w/includes/content/WikitextContent.php(368): Parser->parse(string, Title, ParserOptions, boolean, boolean, integer)
#10 /../w/includes/content/AbstractContent.php(555): WikitextContent->fillParserOutput(Title, integer, ParserOptions, boolean, ParserOutput)
#11 /../w/extensions/SemanticMediaWiki/includes/ContentParser.php(186): AbstractContent->getParserOutput(Title, integer, ParserOptions, boolean)
#12 /../w/extensions/SemanticMediaWiki/includes/ContentParser.php(145): SMW\ContentParser->fetchFromContent()
#13 /../w/extensions/SemanticMediaWiki/src/MediaWiki/Jobs/UpdateJob.php(197): SMW\ContentParser->parse()
#14 /../w/extensions/SemanticMediaWiki/src/MediaWiki/Jobs/UpdateJob.php(137): SMW\MediaWiki\Jobs\UpdateJob->parse_content()
#15 /../w/extensions/SemanticMediaWiki/src/MediaWiki/Jobs/UpdateJob.php(94): SMW\MediaWiki\Jobs\UpdateJob->doUpdate()
#16 /../w/extensions/SemanticMediaWiki/src/SQLStore/Rebuilder/Rebuilder.php(237): SMW\MediaWiki\Jobs\UpdateJob->run()
#17 /../w/extensions/SemanticMediaWiki/src/Maintenance/DataRebuilder.php(427): SMW\SQLStore\Rebuilder\Rebuilder->rebuild(integer)
#18 /../w/extensions/SemanticMediaWiki/src/Maintenance/DataRebuilder.php(349): SMW\Maintenance\DataRebuilder->do_update(integer)
#19 /../w/extensions/SemanticMediaWiki/src/Maintenance/DataRebuilder.php(200): SMW\Maintenance\DataRebuilder->rebuild_all()
#20 /../w/extensions/SemanticMediaWiki/maintenance/rebuildData.php(202): SMW\Maintenance\DataRebuilder->rebuild()
#21 /../w/maintenance/doMaintenance.php(99): SMW\Maintenance\RebuildData->execute()
#22 /../w/extensions/SemanticMediaWiki/maintenance/rebuildData.php(261): require_once(string)
#23 {main}
```

**Comment by mwjames** [comment]
> Call to undefined method ParserOutput::addModuleScripts()

Most likely related to https://github.com/wikimedia/mediawiki/commit/c6f344083271fb2b289c5690bbe2a16ce4a714c5.

**Comment by kghbln** [comment]
> Most likely related to wikimedia/mediawiki@c6f3440.

Thanks for the info! @oetterer FYI

**Comment by oetterer** [comment]
fixed as of #16
